tests/framework/utils/k8s_helper.go

```diff
@@ -395,4 +410,19 @@ func (k8sh *K8sHelper) WaitUntilPodIsDeleted(podNamePattern string) bool{
 		time.Sleep(3 * time.Second)
 	}
 	return false
-}
\ No newline at end of file
+}
+
+func (k8sh *K8sHelper) WaitUntilPVCIsBound(pvcname string) bool {
+
+	inc := 0
+	for inc < 30 {
+		out, _ := k8sh.GetPVCStatus(pvcname)
```

Comment:
when we have retry loops like this it would be good to have log statements so we can tell what is happening. Long delays between logging makes troubleshooting difficult. That can be in another PR though